### ISSUE TITLE
tomb open -p should be the default

### ISSUE THREAD

**Issue by stevelitt** [dyne/Tomb#323]
By default, tomb open makes all files in the tomb user root, group root. In order for tomb open to leave user and group as they were, you need to add the -p option. I think -p should be the default because...

If you *ever* forget to use -p, your files are permanently owned and grouped root. You can't copy them out to another place: They're already root. If you close the tomb and reopen with -p, they're still root. Imagine the sinking feeling you'd have in your stomach if you forgot to use -p. What do you do now? There's no "undo".

If the default were to keep ownerships as-is, and the person wanted all root, and opened without specifying all-root, no harm no foul. They can either close and then reopen as all root, or they can simply do chroot -R and chgrp -R.

Accidentally opening as as-is when what you wanted was all-root is an easily recoverable mistake. But accidentally opening as all-root when what you wanted is as-is is a no-recourse, better-have-recent-backups tragedy.

**Comment by stevelitt** [comment]
In the preceding, what I meant to say was not "root", but "current user". Which is just as much of a disaster.

The following is a patch to  make preserve-ownership default, and require a -P (uppercase P) to not preserve ownership:

https://pastebin.com/jbDcBmAW

The preceding patch creates errors if someone uses -p. If that's a bad thing, the second changed line modified could still contain the "p -preserve-ownership=p", and then right after the first changed line could be: 

option_is_set -p && print "-p now the default and is unneeded. Use -P if you DO NOT want to preserve ownership."

**Comment by stevelitt** [comment]
After a week of working with Tomb, I completely understand Jaromil's comment about real use-cases, and in fact I will be using it without -p for my backups. Not having -p on the first open relieves the need to chown the Tomb Directory.

However, accidentally or mistakenly opening it with -p is a minor annoyance: You close it and and open it without. But if you accidentally or mistakenly open it *with* -p, without even closing or writing to it, you will irretrievably lose all former ownership information. Therefore, I think that the docs and manpage should *PROMENENTLY*  display strong warnings about this, and give advice how to make shellscript substitutes so that anybody who needs such a thing can make themselves a version of Tomb that defaults to -p.

In summary, yeah, for most occasions no -p is best, but docs should warn and advise.

**Comment by jaromil** [comment]
Agreed, there is always room for improving documentation.
